### ISSUE TITLE
Include group in moveResizeValidator

### ISSUE THREAD

**Issue by chriemmy** [namespace-ee/react-calendar-timeline#514]
**Is your feature request related to a problem? Please describe.**
I have a set of rules defined in `moveResizeValidator`. They ensure that certain time ranges cannot be allocated. The rules can be defined fine when I have only one group and the validator works.
But when I move an item to a different group I would like to define different time ranges where an item cannot be placed for that particular group.

**Describe the solution you'd like**
Add new parameter to `moveResizeValidator` function **`group`**. It would contain new group for item which is moved by user.

**Comment by Ilaiwi** [comment]
@chriemmy yes this makes perfect sense. This would be a great first PR for anyone who would like to contribute to this library

**Comment by acemac** [comment]
I did this as a part of #412

**Comment by developer-ocansey** [comment]
Closing as part of issue triage. This issue was created before 2023 and has had no recent activity. If you're still experiencing this on v0.30+, please open a new issue with a minimal reproduction.
